django/contrib/admin/options.py

```diff
@@ -109,6 +109,7 @@ class BaseModelAdmin(six.with_metaclass(forms.MediaDefiningClass)):
     formfield_overrides = {}
     readonly_fields = ()
     ordering = None
+    orderable_by = ()
```

Comment:
What another implementation for controlling fields, allowed for ordering, would you suggest to think about? Thanks.